crypto/fipsmodule/aes/asm/aesv8-armx.pl

```diff
@@ -276,11 +276,12 @@ ()
 
 	vst1.8	{$inout},[$out]
 	ret
-.size	GFp_${prefix}_${dir}crypt,.-GFp_${prefix}_${dir}crypt
+.size	${prefix}_${dir}crypt,.-${prefix}_${dir}crypt
 ___
 }
 &gen_block("en");
-&gen_block("de");
+# Decryption removed in *ring*.
+# &gen_block("de");
```

Comment:
If we don't do this then we'd need to prefix the generated decrypt function. But since we don't need/want it, it's better to just delete it.